config/routes.rb

```diff
@@ -1,12 +1,15 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  devise_for :users
   # Defines the root path route ("/")
   # root "articles#index"
   resources :users do
     resources :posts  # Define the route for the user's posts
```

Comment:
- [ ] [Optional] [Highly recommended] Kindly add `users index` page as your home page (root route). it will be like that:

```ruby
root "users#index"
```